code/game/objects/items/weapons/implants/implant_abductor.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/weapon/implant/abductor
+	name = "recall implant"
+	desc = "Returns you to the mothership."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "implant"
+	activated = 1
+	origin_tech = "materials=2;biotech=7;magnets=4;bluespace=4;abductor=5"
+	var/obj/machinery/abductor/pad/home
+	var/cooldown = 30
+
+/obj/item/weapon/implant/abductor/activate()
+	if(cooldown == initial(cooldown))
+		home.Retrieve(imp_in,1)
+		cooldown = 0
+		processing_objects.Add(src)
+	else
+		to_chat(imp_in, "<span class='warning'>You must wait [30 - cooldown] seconds to use [src] again!</span>")
```

Comment:
This is just moving the file; I haven't made any changes to the implant, explicitly.